static/bidder-info/playdigo.yaml

```diff
@@ -14,3 +14,10 @@ capabilities:
     - banner
     - video
     - native
+userSync:
+  redirect:
+    url: "https://cs.playdigo.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://cs.playdigo.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
```

Comment:
This endpoint does not appear to work. I'm getting a 404. I tested it with the following URL:
`https://cs.playdigo.com/pbserverIframe?gdpr=0&gdpr_consent=CPuESnSPuESnSPiAAAENCZCAAIAAAAAAAAAAAQAAAAAEAAAAAA&ccpa=&gpp=&gpp_sid=&pbserverUrl=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dplaydigo%26gdpr%3D0%26gdpr_consent%3DCPuESnSPuESnSPiAAAENCZCAAIAAAAAAAAAAAQAAAAAEAAAAAA%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3DabcUID`

<img width="1186" alt="Screenshot 2024-08-02 at 10 54 08 AM" src="https://github.com/user-attachments/assets/34f2850d-6a49-492d-bf1f-7c10d8b66195">

Reply:
fixed

Reply:
Confirmed iframe works
<img width="1170" alt="Screenshot 2024-08-06 at 10 41 39 AM" src="https://github.com/user-attachments/assets/727c22cf-95be-47ff-893f-0c84c08b3e05">